lib/thrift/parser/resolver.ex

```diff
@@ -36,6 +36,8 @@ defmodule Thrift.Parser.Resolver do
     new_mappings = local_mappings
     |> Map.new(fn {name, val} ->
       case val do
+        val when is_tuple(val) ->
+          {:"#{include_name}.#{name}", val}
         val when is_atom(val) ->
```

Comment:
Could we also do `val when is_atom(val) or is_tuple(val)` and keep this one case?